worlds/alttp/test/dungeons/TestDungeon.py

```diff
@@ -55,6 +55,6 @@ def run_tests(self, access_pool):
                 for item in items:
                     item.classification = ItemClassification.progression
                     state.collect(item, event=True)  # event=True prevents running sweep_for_events() and picking up
-                state.sweep_for_events()             # key drop keys repeatedly
+                state.sweep_for_advancements()             # key drop keys repeatedly
```

Comment:
That comment used to be col-aligned with the one above, and now it's not anymore.
Personally I don't think aligning comments has much value, and I would instead squeeze it closer to the code on its line, but regardless of preference, the current placement seems wrong

Reply:
Answer for 2: It touched Stardew in a previous iteration, then I force-pushed over that, but still pinged you not realizing I no longer need to. It's my b, sorry <3